build.gradle

```
@@ -57,13 +57,13 @@ allprojects {
         }
 
         maven {
-            url = 'https://maven.wso2.org/nexus/content/repositories/orgballerinalang-1614'
+            url = 'https://repo.maven.apache.org/maven2'
         }
 
         maven {
-            url = 'https://repo.maven.apache.org/maven2'
+            url = 'https://maven.pkg.github.com/ballerina-platform/ballerina-lang'
```

We have to add the credentials here.
```suggestion
            url = 'https://maven.pkg.github.com/ballerina-platform/ballerina-lang'
            credentials {
                username System.getenv("packageUser")
                password System.getenv("packagePAT")
            }
```